histomicstk/segmentation/label/trace_object_boundaries.py

```diff
@@ -117,7 +123,54 @@ def trace_object_boundaries(im_label,
                 im_label, dtype=np.int), conn, x_start, y_start, max_length
         )
 
+        bx, by = _remove_thin_colinear_spurs(bx, by, eps_colinear_area)
+
         X.append(bx)
```

Comment:
Should we skip adding it to the list here if `len(bx) == 0`? 

Or should we let the user of this function handle it?

Reply:
I think we should skip adding it here.  An object with no boundary is no object...

Reply:
Alright, I dropped them!